src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(x, n) {
+      return x + n;
+    },
+
+    subtract(x, n) {
+      return x - n;
+    },
+
+    multiply(x, n) {
+      return x * n;
+    },
+
+    divide(x, n) {
+      return x / n;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, x) {
+      this.result = callback(this.result, x);
```

Comment:
pass just value from operate

```suggestion
    operate(callback, x) {
      callback(x);
```